src/shutdown.cc

```diff
@@ -257,9 +259,15 @@ int main(int argc, char **argv)
     auto shutdown_type = shutdown_type_t::POWEROFF;
 
     const char *execname = base_name(argv[0]);
-    if (strcmp(execname, reboot_execname) == 0) {
+    if (strcmp(execname, halt_execname) == 0) {
+        shutdown_type = shutdown_type_t::HALT;
+    }
+    else if (strcmp(execname, reboot_execname) == 0) {
         shutdown_type = shutdown_type_t::REBOOT;
     }
+    else if (strcmp(execname, soft_reboot_execname) == 0) {
+        shutdown_type = shutdown_type_t::SOFTREBOOT;
+    }
```

Comment:
This is ok but the manual page needs updating.

Reply:
From shutdown.8:

> -s
> Restart the service manager and all user-space services without restarting the system.
> This is the default if executed as soft-reboot.

I don't see what should get updated in this doc.

Reply:
You're right, it's already ok. I was looking at the previous version man page by mistake.